### PR TITLE
Setup project structure for Vet clinic

### PR DESCRIPTION
**Pull request by brainconnect93**
## What I did in this Pull request:

>In this pull request I:

- Setup the project structure according to the provided [template](https://github.com/microverseinc/curriculum-databases-projects-template)
- Created the vet_clinic database
- Created the animal's table in the vet_clinic database
- Populated the animal's table in the vet_clinic database with data
- Queried the animal's table in the vet_clinic database
- Added the application screenshot to the project inside an image folder

kindly review and approve.

## My Screenshots:
vet-clinic
vet-clinic1
vet-clinic2

### CODE REVIEW

**Review by vechicin — COMMENTED** [review]
# Hi @brainconnect93 ✋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😞 

## Invalid Code Review Request ⚠️ 

You have submitted a project, that does not follow Gitflow.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name from animals WHERE name LIKE '%mon%';
```
Comment (addod19) [inline queries.sql]: - [x] It would be better if we change the `%mon%` to `%mon`, this will help us make the right queries

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @brainconnect93, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Github-flow followed :100: 
- [ ] README is descriptive :1st_place_medal: 
- [ ] All queries are support with the screenshots :+1: 
- [ ] Data is added in the data.sql file :rocket: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (brainconnect93) [reply queries.sql]: done as requested

**Review by vechicin — APPROVED** [review]
# Hi @brainconnect93 ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Good job implementing the required changes.

## Optional suggestions 📂 

- [x] You did not follow Gitflow and @addod19 mistakenly reviewed your PR when he should've invalidated it just as I did. For future reference **DO NOT FORGET TO OPEN A DEV BRANCH, MAKE IT THE DEFAULT BRANCH AND THEN MERGE YOUR FEATURE BRANCHES TO THE DEV BRANCH**, otherwise you risk your projects getting invalidated over and over again.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**